seed/seed20.go

```diff
@@ -596,10 +594,65 @@ func (s *seed20) lookupUnassertedComponent(comp20 internal.Component20, info *sn
 	}
 	return Component{
 		Path:         compPath,
-		CompSideInfo: csi,
+		CompSideInfo: *csi,
 	}, nil
 }
 
+func (s *seed20) deriveSideInfo(snapRef naming.SnapRef, modelSnap *asserts.ModelSnap, optSnap *internal.Snap20, handler ContainerHandler, snapsDir string, tm timings.Measurer) (path string, sideInfo *snap.SideInfo, seedComps []Component, err error) {
+	var snapRev *asserts.SnapRevision
+	var snapDecl *asserts.SnapDeclaration
+	path, snapRev, snapDecl, err = s.lookupVerifiedRevision(snapRef, handler, snapsDir, tm)
```

Comment:
You can get rid of some of these explicit declarations since you don't have to worry about shadowing `err` now. I also don't know if the named returns are very useful here.

Reply:
We use named returns in `lookupVerifiedRevision`, and in general when we have many return values, so I think these are still useful